var/spack/repos/builtin/packages/netcdf-fortran/package.py

```diff
@@ -0,0 +1,18 @@
+from spack import *
+
+class NetcdfFortran(Package):
+    """Fortran interface for NetCDF4"""
+
+    homepage = "http://www.unidata.ucar.edu/downloads/netcdf/netcdf-cxx/index.jsp"
+    url      = "http://www.unidata.ucar.edu/downloads/netcdf/ftp/netcdf-fortran-4.4.3.tar.gz"
+
+    version('4.4.3', 'bfd4ae23a34635b273d3eb0d91cbde9e')
+
+    variant('mpi', default=True, description='Enables MPI parallelism')
```

Comment:
@citibeth sounds good. I think it'll be less work to maintain if you can get away with keeping the variants on netcdf and letting the C++ and fortran interfaces figure out the right options based on the netcdf spec.

The thing to remember about variants is that *Spack* sets them all concretely before any install (that's the concretization phase).  So you're guaranteed that any variants on `netcdf` will have defined values by the time `netcdf-fortran` is built.  The idea is that this makes it easier to figure out what to do in `install()` -- the combinatorial cruft and constraint merging is handled by concretization.